Stripe/PublicHeaders/STPConnectAccountAddress.h

```diff
@@ -0,0 +1,69 @@
+//
+//  STPConnectAccountAddress.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 8/2/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPFormEncodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ An address to use with `STPConnectAccountParams`.
+ */
+@interface STPConnectAccountAddress : NSObject <STPFormEncodable>
```

Comment:
how many different address objects can we fit in one SDK?? 🙃

Reply:
😓 wish I could nest this in the `STPConnectAccount` namespace